holidays/locale/en_TC/LC_MESSAGES/TC.po

```diff
@@ -0,0 +1,83 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# Turks and Caicos Islands holidays.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.72\n"
+"POT-Creation-Date: 2025-04-27 22:07-0500\n"
+"PO-Revision-Date: 2025-04-27 22:07-0500\n"
+"Last-Translator: Shalom Donga <shalomdonga@gmail.com>\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_TC\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.6\n"
+"X-Source-Language: en_TC\n"
+
+#. New Year's Day.
+msgid "New Year's Day"
+msgstr ""
+
+#. Commonwealth Day.
+msgid "Commonwealth Day"
+msgstr ""
+
+#. Good Friday.
+msgid "Good Friday"
+msgstr ""
+
+#. Easter Monday.
+msgid "Easter Monday"
+msgstr ""
+
+#. JAGS McCartney Day. Previously National Heroes Day until 2020.
+msgid "JAGS McCartney Day"
+msgstr ""
+
+#. JAGS McCartney Day. Previously National Heroes Day until 2020.
```

Comment:
Running `make l10n` again should take care of the code comment updates for you automatically 👍